src/DynamoCoreWpf/Commands/WorkspaceCommands.cs

```diff
@@ -206,6 +218,27 @@ public DelegateCommand ShowInCanvasSearchCommand
 
         #region Properties for Command Data Binding
 
+        public bool CanCopy
+        {
+            get { return DynamoViewModel.CanCopy(null); }
+        }
+
+        public bool CanPaste
+        {
+            get { return DynamoViewModel.CanPaste(null); }
+        }
+
+        public bool CanCopyOrPaste
+        {
+            get { return CanCopy || CanPaste; }
+        }
+
+        private void Paste(object param)
```

Comment:
I think this method should go on workspaceViewModel like the other methods here to keep the structure parallel. (this is not a property for command data binding)